neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -175,6 +175,7 @@ private void OnGetStateRootsReceived(GetStateRootsPayload payload)
                     break;
                 state_roots.Add(state.StateRoot);
             }
+            if (state_roots.Count == 0) return;
```

Comment:
It should be the same mecause `StateRootsPayload` checks it 😕

Reply:
removed